templates/4a_compensate_image_data.ipynb

```diff
@@ -38,9 +38,8 @@
     "import shutil\n",
```

Comment:
<p>The second paragraph should be up where cohort_name is defined</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>